shared/pages/Wallet/Row/Row.js

```diff
@@ -383,7 +385,7 @@ export default class Row extends Component {
                 <i className="fas fa-sync-alt" styleName="icon" />
                 <span>
                   {
-                    balanceError ? '?' : String(balance).length > 4 ? balance.toFixed(4) : balance
+                    balanceError ? '?' : BigNumber(balance).dp(5, BigNumber.ROUND_CEIL).toString()
```

Comment:
Так же округляет в бОльшую сторону:

![Screenshot_20190612_145907](https://user-images.githubusercontent.com/43396375/59324785-c598ad80-8d23-11e9-8c73-cbcabba41404.png)
![Screenshot_20190612_145924](https://user-images.githubusercontent.com/43396375/59324786-c598ad80-8d23-11e9-9e50-0ad652bb1d7f.png)
![Screenshot_20190612_145938](https://user-images.githubusercontent.com/43396375/59324787-c598ad80-8d23-11e9-8537-6dababf1bac7.png)

Попробуй так:
```suggestion
                    balanceError ? '?' : BigNumber(balance).dp(5, BigNumber.ROUND_FLOOR).toString()
```